app/models/repository.rb

```diff
@@ -33,6 +33,13 @@ def self.license_options
     self.user.decrement!(:reputation, 25)
   end
 
+  #GET VALUES FROM DATE PICKER (MAYBE)
+  # scope :between_dates_picked, ->(start_date , end_date){ where('created_at BETWEEN ? AND ? ', start_date , end_date) }
+
+  # Is this a good approach to generate the url?
```

Comment:
I think you can do ` Rails.application.routes.url_helpers.repository_path(slug: slug, user_username: user.username)`